### DISCUSSION TITLE
Wofür ist das Touch Icon

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#1091] [General]
Hi,

kann mir mal bitte jemand schreiben was es mit dem touch Icon auf sich hat! Ich habe mich durch das Menü gehangelt konnte aber nichts finden.

**Comment by tbnobody** [comment]
Please see #1060

**Comment by jblond4711** [comment]
Danke, das ging an mir vorbei.
Das mit dem Icon funktioniert dann aber nur auf dem Homescreen? Wenn ich die URL unter den Favoriten in Safari speichere ist da leider kein Icon zu sehen.

**Comment by tbnobody** [comment]
Scheint so ja... Wenn du einen Tip hast welcher Meta Tag für den Safari dafür notwendig ist, gerne her damit :)
Mit Chrome (und allen Chrome basierten Browsern wie Vivaldi, Edge usw) klappt es problemlos. Firefox klappt ebenfalls...
